jobs/furnishings/src/furnishings/stage_processors/post_processor.py

```diff
@@ -152,10 +87,124 @@ def get_info_by_name(name: Furnishing.FurnishingName) -> dict:
         return XmlMeta.furnishings[name]
 
 
+class PostProcessor:
+    """Processor after stage run of furnishings job."""
+
+    def __init__(self, app, furnishings_dict):
+        """Create post process helper instance."""
+        self._app = app
+        self._furnishings_dict = furnishings_dict
+        self._processed_date = LegislationDatetime.now()
+
+        # setup the sftp connection objects
+        self._bclaws_sftp_connection = SftpConnection(
+            username=app.config.get('BCLAWS_SFTP_USERNAME'),
+            host=app.config.get('BCLAWS_SFTP_HOST'),
+            port=app.config.get('BCLAWS_SFTP_PORT'),
+            private_key=app.config.get('BCLAWS_SFTP_PRIVATE_KEY'),
```

Comment:
A note on passing in the `private_key` value. [PPR's](https://github.com/bcgov/ppr/blob/main/services/document-delivery-service/src/document_delivery_service/service.py#L54) value of private key is encoded. I guess we will do the similar step, but it depends on how we store the values in 1password.

Reply:
Yeah I'm not sure how PPR was storing the private key value, but when I included the decoding code it was giving me `binascii.Error: Incorrect padding`. Probably just a matter of how I've been saving the private key in the `.env` file. Something to keep in mind though for when we add it to 1pass.

Reply:
I got the same error when I was testing it. I had to base64-encode the private key (though the string is already encrypted) and load it directly from the encoded file. Then the error was gone.

Reply:
Not sure I follow. Are you saying you set the `.env.BCLAWS_SFTP_PRIVATE_KEY ` variable to the private key file path? And also base64-encode the contents of the private key file?

Reply:
No, `BCLAWS_SFTP_PRIVATE_KEY` is the a base64-encoded content of the private key.

I did this, and it worked.
```python
# before it, run `base64 private_key > private_key_base64`
with open (PRIVATE_KEY_PATH, 'rb') as private_key_file:
    private_key = private_key_file.read()

sftp = SftpConnection(
    username=USERNAME,
    host=HOST,
    port=PORT,
    private_key=base64.b64decode(private_key).decode('utf-8'),
    private_key_algorithm=PRIVATE_KEY_ALGORITHM
)
```

Reply:
hmmm...I think so, but it depends. We can leave it for now. Once we get the key and other configs setup, we can go back to make a change.